plugins/modules/vmware_deploy_ovf.py

```diff
@@ -105,9 +109,17 @@
     ovf:
         description:
         - 'Path to OVF or OVA file to deploy.'
+        - This is a required parameter, if C(ovf) is not set and C(url) parameter must be set.
+        - C(ovf) and C(url) are mutually exclusive parameters.
         aliases:
             - ova
         type: path
+    url:
+        description:
+        - 'URL for OVA file to deploy.'
+        - This is a required parameter, if C(url) is not set and C(ovf) parameter must be set.
+        - C(url) and C(ovf) are mutually exclusive parameters.
+        type: str
```

Comment:
```suggestion
        type: str
        version_added: '3.9.0'
```

We should document when this parameter has been added.

Let's hope the CI doesn't make it too hard for us this time :unamused: